public/samples/Automation/tutorials/ExampleImplementation.sol

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/**
+ * THIS IS AN EXAMPLE CONTRACT THAT USES HARDCODED
+ * VALUES FOR CLARITY.
+ * THIS IS AN EXAMPLE CONTRACT THAT USES UN-AUDITED CODE.
+ * DO NOT USE THIS CODE IN PRODUCTION.
+ */
+
+/**
+ * THIS IS AN EXAMPLE OF A CONTRACT FOR CIRCUIT BREAKER. IT HAS 2 PRIMARY FUNCTIONS: (A) TO CHECK FOR A CERTAIN CONDITION AND (B) TO EXECUTE AN ACTION
+ * THIS IS A FLEXIBLE CONTRACT THAT CAN BE USED TO BUILD ANY CONDITION SET AND ANY EXECUTION. IN THIS CASE THE CONDITION BEING CHECKED IS THE PRICE OF AN ASSET
+ * AND THE ACTION BEING EXECUTED IS INCREMENTING A COUNTER. THIS CONTRACT BECOMES AN INPUT INTO AN UPKEEP CONTRACT FOR MANY DIFFERENT CIRCUIT BREAKING
+ * CONDITIONS. SO THIS WAY, THE UPKEEP CONTRACT REMAINS THE SAME, BUT USING CHECK DATA YOU CAN PASS IN A CONTRACT LIKE THIS AS A CUSTOMIZABLE CIRCUIT BREAKER
+ *
+ */
+
+contract ExampleDataConsumerV3 is Ownable {
+    AggregatorV3Interface internal dataFeed;
+
+    uint256 public counter;
+    int public maxbalance;
+    bool emergencyPossible;
+    address feedAddress;
+    event emergencyActionPerformed(uint256 counter, address feedAddress);
+    bool circuitbrokenflag;
+    int currentAnswer;
+
+    /**
+     * Network: Sepolia
+     */
+    constructor(
+        int _maxbalance,
+        address _feedAddress,
+        bool _emergencyPossible
+    ) {
+        dataFeed = AggregatorV3Interface(_feedAddress);
+        maxbalance = _maxbalance;
+        counter = 0;
+        emergencyPossible = _emergencyPossible;
+        feedAddress = _feedAddress;
+    }
+
+    /**
+     * Returns true if the price is greater than a certain value.
+     */
+    function isFeedParamMet() public view returns (bool) {
+        // prettier-ignore
+        return ((getLatestData() > maxbalance) && !circuitbrokenflag);
+    }
```

Comment:
FYI here's where I changed the code to resolve a compiler warning in Remix

Reply:
It's fine to leave the warnings in. Errors are blockers.